src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
+$card-size: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+
+  grid-template-columns: repeat(4, $card-size);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media  (max-width: 1024px) {
```

Comment:
It is better to use one state Mobile First or Desktop First (recommend to use Mobile one) 
[link](https://ishadeed.com/article/the-state-of-mobile-first-and-desktop-first/)
Update your media-queries using `min-width`